src/dimred/pca/config.vsh.yaml

```diff
@@ -25,6 +25,11 @@ functionality:
       default: "rna"
       required: false
 
+    - name: "--layer"
```

Comment:
Should the layer parameter be defined in workflows/integration?

Reply:
Added!